addon/index.js

```diff
@@ -93,7 +98,15 @@ export function changeset(obj, validateFn = defaultValidatorFn, validationMap =
      * @return {Any}
      */
     setUnknownProperty(key, value) {
-      return this._validateAndSet(key, value);
+      let changesetOptions = get(this, OPTIONS);
+      let skipValidationsOnSet = get(changesetOptions, 'skipValidationsOnSet');
+      
+      if (skipValidationsOnSet) {
+        return this._setProperty(true, { key, value });
+      }
+      else {
```

Comment:
This `else` block is unnecessary, just `return this._validateAndSet(key, value);` is sufficient